nupic/research/TP_shim.py

```diff
@@ -27,6 +27,9 @@
 import numpy
 
 from nupic.research.temporal_memory import TemporalMemory
+from nupic.research.monitor_mixin.temporal_memory_monitor_mixin import (
+  TemporalMemoryMonitorMixin)
+class MonitoredTemporalMemory(TemporalMemoryMonitorMixin, TemporalMemory): pass
```

Comment:
Actually, on second thought, this isn't worth it because it's a mixin class, so you should be able to mix it in to any `TemporalMemory` instance, either `TemporalMemory` (python), `FastTemporalMemory`, or `TemporalMemory` (C++). I think it's better to let the client of the mixin make the decision on which one to use.